src/MainTab.py

```diff
@@ -218,11 +218,11 @@ def loadKernelModules():
         Load kernel modules to interact with CAN networks (``can`` and ``vcan``).
         """
 
-        cmds = ["modprobe can", "modprobe vcan"]
-        for cmd in cmds:
-            process = subprocess.Popen(
-                cmd.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            output, error = process.communicate()
+        cmds = '''
+            modprobe can
```

Comment:
can you make this a single line so it doesn't end up as multiple lines in the logging window?

Reply:
Done.